etcdctl/ctlv3/command/auth_command.go

```diff
@@ -48,7 +49,21 @@ func authEnableCommandFunc(cmd *cobra.Command, args []string) {
 	}
 
 	ctx, cancel := commandCtx(cmd)
-	_, err := mustClientFromCmd(cmd).Auth.AuthEnable(ctx)
+	cli := mustClientFromCmd(cmd)
+	var err error
+	for err == nil {
+		if _, err = cli.AuthEnable(ctx); err == nil {
+			break
+		}
+		if err == rpctypes.ErrRootRoleNotExist {
```

Comment:
It's not creating a root user, just the root role and adding the root user to the root role. Should it still print some notification that it's auto-creating?